templates/kubernetes/terraform/modules/kubernetes/backend-service.tf

```diff
@@ -0,0 +1,17 @@
+data "aws_secretsmanager_secret_version" "cf_keypair" {
```

Comment:
Mind changing the name of the file to `backend_service.tf` with an underscore? You probably based this off `database-service.tf` which I've been meaning to rename because it's the only one that uses a dash. 🤦

Reply:
Actually if you feel like renaming the database one too, that would be nice! 😁

Reply:
Sure!